src/main/resources/default/templates/biz/process/process-output-chart.html.pasta

```diff
@@ -16,17 +16,17 @@
 
         <script type="text/javascript">
             @raw {
-                $(document).ready(function () {
+                sirius.ready(() => {
                     var chartData = @chartOutput.parseChartData(chart);
                     if (chartData.type === 'line') {
                         lineChart('#chart-@id', chartData.labels, chartData.lines);
                     } else if (chartData.type === 'bar') {
                         barChart('#chart-@id', chartData.labels, chartData.bars);
                     } else if (chartData.type === 'polarArea') {
-                        $('#chart-@id').attr('height', 200);
+                        document.querySelector('#chart-@id').height = 200;
                         polarAreaChart('#chart-@id', chartData.labels, chartData.data);
                     } else if (chartData.type === 'dougnut') {
-                        $('#chart-@id').attr('height', 180);
+                        document.querySelector('#chart-@id').height = 180;
                         doughnutChart('#chart-@id', chartData.labels, chartData.data, {
                             maintainAspectRatio: false
                         });
```

Comment:
I could not test this code, no process with chart output in OX

Reply:
Isnt this one of them? https://oxomi.test.scireum.com/job/clickhouse-size

Reply:
Many thanks for the pointer. However, as far as I understand it, this one works differently and uses a different template.

Apparently, like tables, chart output can be attached as output to processes via `ProcessContext.addCharts(…)`. In this case, the template here should be used. I did not find any use of this method in OX, though. In order to test this, I fear we would need to write a dummy job – or hope that another of our products actually uses it.